src/components/Button.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import { get_current_component as getComponent } from "svelte/internal";
```

Comment:
huh, odd that it's underscore_style not camelCaseStyle, looks like it's the only function like that in there... thanks for `as getComponent` :)

Reply:
Yes, this is as svelte style difference. Lots of camel-case stuff for the deeper internals. Most other stuff is single-word though.